ci/scripts/run-fpga-cw310-tests.sh

```diff
@@ -40,6 +40,10 @@ export BITSTREAM="--offline --list ${SHA}"
 # in case we've crashed the UART handler on the CW310's SAM3U
 trap 'ci/bazelisk.sh run //sw/host/opentitantool -- --interface=cw310 fpga reset' EXIT
 
+# In case tests update OTP or otherwise leave state on the FPGA we should start
+# by loading a bitstream to clear state
+ci/bazelisk.sh run //sw/host/opentitantool -- --interface=cw310 fpga load-bitstream "${BIT_CACHE_DIR}/${BIT_NAME_PREFIX}.orig"
```

Comment:
Whoops, I commented on an orphaned commit a second ago. This is what I said:

> We should probably tack on --rcfile= here and on line 41.

Reply:
You're right...